include/exadg/operators/operator_base.cpp

```diff
@@ -1707,7 +1707,7 @@ OperatorBase<dim, Number, n_components>::cell_loop_calculate_system_matrix(
       else
         cell_v->get_dof_indices(dof_indices);
 
-      if(!is_dg)
+      if(not(is_dg))
```

Comment:
here it is again even though the `is_dg` is super simple ...
Somehow I think we are just a bit relaxed with the `not(...)`, are we? Totally fine for me, just spotted that :laughing: